polaris.shopify.com/src/components/IconsPage/IconsPage.tsx

```diff
@@ -1,310 +1,145 @@
-import { useState, useRef, useCallback } from "react";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import Link from "next/link";
 import { Dialog } from "@headlessui/react";
-import Fuse from "fuse.js";
```

Comment:
As an ESL, I do like the fuzzy search logic that fuse brings. I never know if it's "access-a-bility" or "access-i-bility".